src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
You're linking to an SCSS file directly in your HTML. Browsers can't interpret SCSS, they only understand CSS. Ensure to compile your SCSS into CSS and link the CSS file here instead.